2019/d04p1Passcodes.d

```diff
@@ -0,0 +1,33 @@
+bool meetsCriteria(int input) {
```

Comment:
On Compiler Explorer, [LDC is doing its thing again](https://godbolt.org/z/8hEpNB)! `imul    rdx, rax, 1717986919` indeed!